src/components/graphs/linegraph/LineGraph.js

```diff
@@ -2,6 +2,8 @@ import React, { useEffect, useState } from 'react';
 import { Line } from 'react-chartjs-2';
 import { options } from '../assets';
 
+import './LineGraph.css';
```

Comment:
The change above this shows that LineGraph.CSS was deleted, but then its being imported here.

Reply:
Yeah, I already changed the import to './LineGraph.less'